Cargo.toml

```diff
@@ -19,14 +19,20 @@ targets = ["aarch64-pc-windows-msvc", "i686-pc-windows-msvc", "x86_64-pc-windows
 widestring = "0.4"
 log = "0.4"
 rand = "0.8"
-bitflags = "1.3"
+bitflags = "2"
 
-libloading = "0.7"
+libloading = "0.8"
 ipnet = "2.3"
-winapi = { version = "0.3", features = ["ws2def", "netioapi", "nldef"] }
+windows-sys = { version = "0.52", features = [
+    "Win32_Foundation",
+    "Win32_Networking",
+    "Win32_Networking_WinSock",
+    "Win32_NetworkManagement_IpHelper",
+    "Win32_NetworkManagement_Ndis"
+]}
 
 [dev-dependencies]
 env_logger = "0.9"
 base64 = "0.13"
-boringtun = "0.3"
+x25519-dalek = { version = "=2.0.0-rc.3", default-features = false, features = ["static_secrets", "getrandom"] }
```

Comment:
Any reason we cant use `2.0.1`?

Reply:
I think no.  It’s just that where I use x25519-dalek there is a temporary limitation for this particular version, which I haven’t gotten around to fixing yet.
I'll replace it with 2.0.1 today.

Reply:
There was either this or a similar error: https://github.com/nulldotblack/wireguard-nt/actions/runs/10126534546/job/28167328669?pr=15#step:3:117